src/plugins/layout.ts

```diff
@@ -411,5 +334,38 @@ export default class Layout implements IPlugin {
       });
     }
 
+    return events;
+  }
+
+  private ensureConsistency(lastAction: string): void {
+    if (config.ensureConsistency) {
```

Comment:
Changed method name to 'checkConsistency'

---
In reply to: [132224906](https://github.com/Microsoft/clarity-js/pull/25#discussion_r132224906) [](ancestors = 132224906)